Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11998,21 +11998,25 @@ void WebPageProxy::requestPointerLock()
 
 void WebPageProxy::didAllowPointerLock()
 {
-    ASSERT(m_isPointerLockPending && !m_isPointerLocked);
-    m_isPointerLocked = true;
-    m_isPointerLockPending = false;
+    if (m_isPointerLockPending) {
```

Comment:
nit: an early return if there is not lock pending would decrease indentation.  Same in WebPageProxy::didDenyPointerLock

Reply:
Done